src/imitation/scripts/parallel.py

```diff
@@ -24,17 +27,30 @@ def parallel(inner_experiment_name: str,
     inner_experiment_name: The experiment to tune. Either "expert_demos" or
       "train_adversarial".
     search_space: `config` argument to `ray.tune.run()`.
+    base_named_configs: `search_space["named_configs"]` is appended to this list
+      before it is passed to the inner experiment's `run()`. Notably,
+      `base_named_configs` doesn't appear in the automatically generated
+      Ray directory name.
+    base_config_updates: `search_space["config_updates"]` is applied to this
+      dict before it is passed to the inner experiment's `run()`.
     upload_dir: `upload_dir` argument to `ray.tune.run()`.
   """
-  trainable = _ray_tune_sacred_wrapper(inner_experiment_name)
+  trainable = _ray_tune_sacred_wrapper(inner_experiment_name,
+                                       base_named_configs,
+                                       base_config_updates)
+  ray_loggers = (tune_logger.JsonLogger,)  # Prevent unused Tensorboard write.
```

Comment:
Make comment more explicit: does Ray write TensorBoard logs by default? 

For that matter, why do we want any Ray Tune logs -- I don't think we report any metrics? Is this to future-proof it in case we start using that feature?

Reply:
It was only annoying to have TB logs because they would clutter up the runs dashboard in Tensorboard.

The JsonLogger seems harmless to include right now.

Reply:
Here's an example JSON log:
```
$ cat result.json
{"done": true, "time_this_iter_s": 44.65022039413452, "timesteps_total": null, "episodes_total": null, "training_iteration": 1, "experiment_id": "cead010016b74119b7d941f19f55a728", "date": "2019-09-18_17-54-05", "timestamp": 1568854445, "time_total_s": 44.65022039413452, "pid": 11013, "hostname": "astar", "node_ip": "169.229.222.160", "config": {"config_updates": {"init_rl_kwargs": {"learning_rate": 0.0003, "nminibatches": 16}, "seed": 1}, "named_configs": []}, "time_since_restore": 44.65022039413452, "timesteps_since_restore": 0, "iterations_since_restore": 1, "trial_id": "dea0dbde"}
```

Reply:
Ended up removing JSON logs too because for the most part they are a subset of Sacred logs